src/pylint_datetime/__init__.py

```diff
@@ -1,4 +1,164 @@
 # License: MIT
 # Copyright © 2023 Frequenz Energy-as-a-Service GmbH
 
-"""checker to ensure correct datetime usage with aware objects in datetime module."""
+"""Pylint checker ensure correct usage of datetime module, especially around aware/naive objects.
+
+Function calls to fromisoformat, strptime and fromisoformat can't be checked - they parse strings.
+If the specified timezone is None, still allowed although they produce naive objects.
```

Comment:
It says 2 times `fromisoformat`, I guess it is an unintentional duplicate. I suggest also making some more explicit clarifications about `None` and add links to the functions.

```suggestion
Function calls to [`datetime.fromisoformat`][datetime.datetime.fromisoformat] /
[`time.fromisoformat`][datetime.time.fromisoformat] and
[`datetime.strptime`][datetime.datetime.strptime] /
[`time.strptime`][datetime.time.strptime] can't be checked - they parse strings. 

[`datetime.combine`][datetime.datetime.combine] also can't be checked - it
combines a `date` and a `time` and will use the timezone from `time` if present,
so we can't be sure if an explicit `tzinfo` argument is needed or not unless we
can inspect the `time` object.

This can't check either if an explicitly specified timezone is `None` (either literally
or because a variable bound to `None` is passed), so in those cases no errors will
be produced.
```